zuul.d/jobs.yaml

```diff
@@ -308,8 +308,10 @@
       - name: github.com/ansible/ansible
       - name: github.com/ansible-collections/cisco.asa
     timeout: 9000
-    irrelevant-files:
+    irrelevant-files: &documentation-files
```

Comment:
I think we could move this into ansible-network-appliance-base job to cut down on the copy / pasta